bigcache_test.go

```diff
@@ -203,6 +204,39 @@ func TestOnRemoveCallback(t *testing.T) {
 	assert.True(t, onRemoveInvoked)
 }
 
+func TestOnRemoveFilter(t *testing.T) {
```

Comment:
Need to add another test that would cover `Expired` as well. Or implement `Expired` in this test. :)